app/Http/Controllers/SitemapController.php

```diff
@@ -12,7 +12,7 @@
 
 class SitemapController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->